### PR TITLE
azure native v2 docs

### PR DESCRIPTION
**Pull request by sean1588**
**Add azure-native v2 docs**

This PR is not ready to be merged yet, as we are still making changes to the files here and need to resolve some build issues when it comes to generating the api docs for the v2 version of this package.

fixes https://github.com/pulumi/registry/issues/2569

### CODE REVIEW

**Comment by susanev** [comment]
@sean1588 wheres the change for adding the note in the card?

here's the updated text for that...
>Azure Native v2 is in preview. Use [Azure Native GA] for a fully supported version.

**Review by susanev — APPROVED** [review]

**Review by susanev — APPROVED** [review]

File: themes/default/content/registry/packages/azure-native-v2/installation-configuration.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+## Installation
+
+The Azure Native provider is available as a package in all Pulumi languages:
+
+* JavaScript/TypeScript: [`@pulumi/azure-native`](https://www.npmjs.com/package/@pulumi/azure-native/v/2.0.0-alpha.1)
+* Python: [`pulumi-azure-native`](https://pypi.org/project/pulumi-azure-native/2.0.0a1/)
+* Go: [`github.com/pulumi/pulumi-azure-native/sdk/go/azure`](https://github.com/pulumi/pulumi-azure-native/releases/tag/v2.0.0-alpha.1)
+* .NET: [`Pulumi.AzureNative`](https://www.nuget.org/packages/Pulumi.AzureNative/2.0.0-alpha.1)
+* Java: `com.pulumi.azurenative` [Pre-release unavailable]
```
Comment (susanev) [inline themes/default/content/registry/packages/azure-native-v2/installation-configuration.md:19]: @mnlumi, thoughts on removing this line entirely?

**Review by susanev — APPROVED** [review]

File: themes/default/content/registry/packages/azure-native-v2/installation-configuration.md (hunk 1 above)
Comment (susanev) [reply themes/default/content/registry/packages/azure-native-v2/installation-configuration.md:19]: @sean1588 we dont have to block merging on this comment

Comment (mnlumi) [reply themes/default/content/registry/packages/azure-native-v2/installation-configuration.md:19]: I was debating removing it, but didn't want anyone to infer that Java would never be available for v2 so I left it and called out that the pre-release is unavailable. I imagine people that use Java would expect this.

**Pull request merged**